content/docs/user-guide/external-dependencies.md

```diff
@@ -54,11 +54,14 @@ $ dvc run -n download_file
 
 ```dvc
 $ dvc run -n download_file
-          -d ssh://user@example.com:/home/shared/data.txt \
+          -d ssh://user@example.com/abs/path/to/data \
           -o data.txt \
-          scp user@example.com:/home/shared/data.txt data.txt
+          scp user@example.com/abs/path/to/data data
 ```
 
+> ⚠️ Please notice `/abs/path/to/data` is an absolute path from the SFTP root
+> (not always configured to be the system root).
```

Comment:
Althouogh... TBH Idk if we can call this "absolute". Isn't it relative to the SFTP root? (which may or may not be the system root, in which case this would be an abs path indeed)

Reply:
Yes, as noted in https://github.com/iterative/dvc.org/pull/1649#discussion_r463982286

Reply:
OK I updated the note... But it's confusing now 🤔🧠